backend/.env.production

```diff
@@ -6,6 +6,8 @@ ENVIRONMENT=production
 PROJECT_NAME="Districtr v2 backend"
 BACKEND_CORS_ORIGINS="https://districtr-v2-frontend.fly.dev,https://districtr-v2-api.fly.dev" # todo
 SECRET_KEY={fill-me}
+LOAD_GERRY_DB_DATA=false
+GPKG_DATA_DIR=sample_data
```

Comment:
H: Don't think we will need these in the prod env file, right?

Reply:
Ah, correct. This is a mistake